### ISSUE TITLE
Dashboard - Filters and data for "Previously Regsitered" 

### ISSUE THREAD

**Issue by DaveEade49** [younginnovations/iatipublisher#1266]
See screenshot below - No organisations are coming up when the Filter "new to publisher" is used.

![image](https://github.com/younginnovations/iatipublisher/assets/110898894/09f2e938-e050-43ff-ab0c-fa45e534a343)

**Comment by praweshsth** [comment]
There are data for only existing organisations in the publisher. There are no organisation who have registered new from the tool.

**Comment by DaveEade49** [comment]
I don't think that is right - Sustainable Livelihoods Foundation is one example where they have registered at the same time

**Comment by DaveEade49** [comment]
Can you let me know what the logic is that's used to determine?

I wonder if you are trying to match date and timestamp, in which case there may be a few seconds delay....

**Comment by sriza** [comment]
@DaveEade49 

The organizations are determined to be new and existing based on how they are created at IATI Publisher i.e., the form they choose to register through.

Based on your information regarding Sustainable Livelihoods Foundation, I looked into our audit log and updated the organizations that were newly created in the publisher tool. It seems there was an issue with the way we updated the method of registration for organizations. The organizations created hereafter will correctly represent the way of creation.

Please have a look at the changes and let me know if the issue persists.

Thank you

**Comment by DaveEade49** [comment]
Thanks, I've looked and this seems to be working ok now.
